straxen/url_config.py

```diff
@@ -229,3 +230,8 @@ def format_arg(arg: str, **kwargs):
     '''apply pythons builtin format function to a string
     '''
     return arg.format(**kwargs)
+
+
+@URLConfig.register('interpolatingmap')
```

Comment:
maybe semantics, but I always thought it it was interpolation map, but maybe, ìtp_map` is even better to keep stings a bit more readable? I don't think it matters much but I just want to ask it now since changing this function later is kind of defeating the purpose of URL configs since these roadmaps to the values shouldn't change.

So better ask such a question now when we introduce the method 😄

Reply:
I also like itp_map better as the protocol name, that way its also clear that its using the methods defined in itp_map.py

Reply:
Sure, that's fine with with me.